loader/src/injector/hook.cpp

```diff
@@ -231,19 +238,22 @@ DCL_HOOK_FUNC(int, pthread_attr_setstacksize, void *target, size_t size) {
     int res = old_pthread_attr_setstacksize((pthread_attr_t *)target, size);
     LOGV("Call pthread_attr_setstacksize in [tid, pid]: %d, %d", gettid(), getpid());
 
-    if (!enable_unloader)
+    if (!enable_unloader) {
```

Comment:
One line ifs don't use {} in our coding style, unless it's  big.

Reply:
Compilers learned a long way from there. Nowadays they will warn upon such case, so that we don't have to worry about that.

Reply:
Compilers are not foolproof; removing these `{}` will greatly increase the odds of user error, despite the compiler's warnings. Constantly reviewing contributer's code (which may contain this and other blunders) is a tiresome process and people _will_ make mistakes.
Having brackets removes the room for doubt altogether.

I'm insisting on this just this once, but if you insist further, I'll revert it.